docker/docker-compose.override.yml.default

```diff
@@ -49,6 +49,9 @@ services:
       - 'traefik.port=8025'
       - 'traefik.frontend.rule=Host:mail-${MAIN_DOMAIN_NAME}'
```

Comment:
Host still needs work to figure, this option does not work with traefik 2 and probably we can use `${COMPOSE_PROJECT_NAME}` for that